core/src/main/scala/com/metamx/tranquility/druid/DruidBeamMaker.scala

```diff
@@ -88,15 +84,15 @@ class DruidBeamMaker[A: Timestamper](
           "format" -> "json",
           "timestampSpec" -> timestampSpec,
           "dimensionsSpec" -> rollup.dimensions.specMap
-        ).asJava
-      ).asJava,
-      "metricsSpec" -> rollup.aggregators.toArray,
+        )
+      ),
+      "metricsSpec" -> Jackson.parse[Seq[Dict]](druidObjectMapper.writeValueAsBytes(rollup.aggregators.toArray)),
```

Comment:
What does this change affect? (What happens without the Jacksonparse and writeValueAsBytes?)